plugins/com.google.cloud.tools.eclipse.appengine.facets/plugin.xml

```diff
@@ -10,7 +10,7 @@
     <project-facet-version facet="com.google.cloud.tools.eclipse.appengine.facet" version="1">
        <constraint>
           <and>
-            <requires facet="jst.web" version="2.5"/>
+            <requires facet="jst.web" version="[2.2-2.5]"/>
```

Comment:
why do we want 2.2 through 2.4? App Engine standard primarily supports 2.5 these days.